app/workers/domain_events_consumer.rb

```diff
@@ -1,22 +1,24 @@
 class DomainEventsConsumer
   include Shoryuken::Worker
 
-  shoryuken_options queue: ENV.fetch('DOMAIN_EVENTS_SQS_QUEUE_NAME', 'undefined'), auto_delete: true
+  shoryuken_options queue: ENV.fetch('DOMAIN_EVENTS_SQS_QUEUE_NAME', 'undefined'),
+                    auto_delete: true,
+                    retry_intervals: [15.minutes, 1.hour]
```

Comment:
So this can't go into a config file? We could always load if from a config file ourselves